keyboards/nullbitsco/nibble/remote_kb.c

```diff
@@ -0,0 +1,128 @@
+#include "remote_kb.h"
+
```

Comment:
Lets have some documentation here -- I had no idea what `remote_kb` is.
Perhaps rename the file to something that matches the actual hardware, along with the exported APIs?